### DISCUSSION TITLE
DARC whitepaper is officially released!

### DISCUSSION THREAD

**Discussion by lidangzzz** [Project-DARC/DARC#89] [Announcements]
Here it is:

[https://github.com/Project-DARC/DARC-whitepaper](https://github.com/Project-DARC/DARC-whitepaper)

**Comment by Lissonwong** [comment]
0x1e81801b5f951e12809ab55e0b916e5e0ee3d107


立党 Lidang ***@***.***>于2024年3月14日 周四13:06写道：

**Reply by qiaoyuanjun** [reply]
立党疯了？邮件停不下来了
